app/assets/stylesheets/_shirts.scss

```diff
@@ -0,0 +1,21 @@
+div.shirt-banner {
+  margin-bottom: $base-spacing;
+  overflow-x: hidden;
+  white-space: nowrap;
+
+  a {
+    @include animation(marquee 20.0s linear infinite);
+    color: white;
+    display: inline-block;
+    font-style: italic;
+    font-weight: bold;
+    padding-left: 100%;
+    text-decoration: none;
```

Comment:
I am mostly pro-no underline because it looks cleaner. As long as there some other way to give affordance to the links. Maybe we could make all links white and change headers to a different color?